tests/integration/jit/tracing/skipif.inc

```diff
@@ -0,0 +1,10 @@
+<?php
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+if (version_compare(PHP_VERSION, "8.0", "<")) {
+  die("skip: PHP < 8.0.0 not supported\n");
+}
```

Comment:
```suggestion
}
if (!extension_loaded('Zend OPcache')) {
  die("skip: Zend OPcache extension required\n");
}
```

Reply:
I made this change in commit [b2a47c](https://github.com/newrelic/newrelic-php-agent/pull/583/commits/b2a47cf5a9fec76c1557d24f184983559db66fef).

After ensuring ```opcache.so``` was loaded the following tests now fail (pass w/o opcache):

tests/integration/jit/tracing/test_span_class_function.php
tests/integration/jit/function/test_span_events_are_created_upon_caught_exception.php